fireant/slicer/pagination.py

```diff
@@ -0,0 +1,19 @@
+class Paginator(object):
+    def __init__(self, limit=0, offset=0, order=()):
+        """
+        Class for keeping track of pagination parameters
+
+        Args:
```

Comment:
I have nothing against google doc style, but mixing styles in a project does not feel right to me.